content/blog/orderly-1.0.1.md

```diff
@@ -0,0 +1,48 @@
+---
+author: "Rich FitzJohn"
+date: 2019-10-18
+title: orderly 1.0.1 released to CRAN
+---
+
+We are pleased to announce the first public release of [`orderly`](https://vimc.github.io/orderly), our reproducible reporting framework, implemented as an R package and now available on CRAN.
+
+The `orderly` package was designed to help with a common pattern in reporting where a report might be run multiple times (over say a number of weeks or months) and where the inputs to the report might change.  In our case at [VIMC](https://www.vaccineimpact.org) this is analysis of impact of vaccines, but this pattern exists in many fields.  When reports are run multiple times they will inevitably vary, but we want to understand **why** they vary - was it a change in the analysis code, in the input data, or in some other dependency of the report?  Critically, we want it to always be completely clear that a given set of inputs belongs to a given set of outputs, without relying on any discipline from the end-user.
+
```

Comment:
The first time I read this, I wondered about being a little bit more clear in terminology about what a *report* is here - is it the code we run, or is it the output of running that code - and whether it would be more clear to talk about, eg, report code staying the same, but report outputs differing, and we want to understand why they vary...

But then I think when you ask "is it the analysis code" that changes, it became more clear that the report code is not as "constant" as I had first assumed, and you're talking about reports changing in a more fluid way. So this is probably fine, but I thought I'd record my thoughts on it anyway, rather than deleting the comment!